### PR TITLE
Release candidate for Luos 1.3.0

### PR DESCRIPTION
**Pull request by nicolas-rabault**

### CODE REVIEW

File: src/pages/embedded/containers/use-profiles.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+To actually create the container template on the Luos network you have to call the creation function specific to your template
+```c
+container_t *TemplateServoMotor_CreateContainer(CONT_CB callback, template_servo_motor_t *var, const char *default_alias, revision_t revision);
+```
+
+The returned `container_t*` is a container structure pointer that will be useful to make your container act in the network after this initialization.
+
+ **callback** is a pointer to the same callback function described on [the container management section](./create-containers.html#how-to-create-and-initialize-a-container).
+ The big difference between templated container and custom containers is that you don't need to manage any message in this callback, because the profile handle it for you.
```
Comment (Simonbdy) [inline src/pages/embedded/containers/use-profiles.md:44]: Is "templated" what you wanted to use or do you mean "template container"? The occurrence appears in line 54 as well.
In any case, template is not a verb so I suggest another form.

File: src/pages/embedded/dev_env/general-integration-consideration.md [hunk 2]
```diff
@@ -10,20 +10,34 @@ To make it work in your environment, you have to:
...
 | NBR_NAK_RETRY | 10 | Number of retries to send after a received NAK. |
 | MAX_CONTAINER_NUMBER | 5 | Number of containers in the node (memory optimisation). |
 | MSG_BUFFER_SIZE | 3*size_msg | Message buffer size. Max size of a message (3 * (7 bytes header + 128 bytes data + 2 bytes CRC)). |
-| MAX_MSG_NB | 2*MAX_CONTAINER_NUMBER | Max number of messages for a container that can be buffered. |
+| MAX_MSG_NB | 2*MAX_CONTAINER_NUMBER | Max number of messages for a container that can be referenced. |
 | NBR_PORT | 2 | Number of PTP on the node ( max 8). See [electronic design](../hardware_topics/electronic-design.md) page.|
+
+You will find the default config for Luos Library in the file <a href="https://github.com/Luos-io/Luos/tree/master/Robus/inc/config.h" target="_blank">config.h</a>,
+
+- Put all your custom configuration for Luos HAL library to modify Luos HAL config and make it fit with your design
```
Comment (Simonbdy) [inline src/pages/embedded/dev_env/general-integration-consideration.md:34]: The verb put doesn't seem to be properly used: "put something _somewhere_". The _somewhere_ part is missing in all these sentences, so I suggest either adding it (where do we put the custom configuration?) or replace the verb put by another one like "get" or "use".

File: src/pages/embedded/tools/gate.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+| Parameters | Defaults value | Description |
+| :---: | :---: | :---: |
+| GATE_BUFF_SIZE | 1024 | Maximum size of 1 formatted Data.|
+| GATE_POLLING | NOT DEFINED | No autorefresh always ask data (more intensive to Luos bandwidth.) |
+| NODETECTION | NOT DEFINED | Gate do not make detection a power up |
+
+If you have an App container on your device managing detections you should define **NODETECTION** avoiding useless detection from the Gate at boot.
+
+If you have an App container on your device using auto-update you should define **GATE_POLLING** avoiding gate to take the lead on the containers your App is using.
```
Comment (Simonbdy) [inline src/pages/embedded/tools/gate.md:73]: Why say "Gate" or "Pipe" instead of "the Gate" or "the Pipe"?

**Review by Simonbdy — CHANGES_REQUESTED** [review]
I asked questions mainly for comprehension, no big changes are requested.
I will then commit my final review later today with minor syntax and typo fixes.

File: src/pages/embedded/containers/msg-handling.md [hunk 4]
```diff
@@ -130,3 +130,79 @@ Luos_SendMsg(app, &msg);
...
+
+|Configuration|execution type|
+|:---:|:---:|
+|[Callback (default)](#Callback-configuration)|runtime callback|
+|[Polling](#polling-configuration)|no callback|
+
+The following sections detail how the different configurations work.
+
+## Callback configuration
+This configuration is the default and most common setup. In this configuration, Luos directly calls the container callback during runtime. The time between the physical reception of a message and the callback may vary depending on the `luos_loop()` function call frequency.<br/>
```
[the remark this answers is not in the document]
Comment (nicolas-rabault) [reply src/pages/embedded/containers/msg-handling.md:148]: Luos_loop is called frequently (depending on the time needed to execute user services). The time between luos_loop execution can be added to the latency needed to get a message from the callback.
Is it better?
```suggestion
This configuration is the default and most common setup. In this configuration, Luos directly calls the container callback during runtime. The time between the physical reception of a message and the callback may vary depending on the `luos_loop()` function execution frequency.<br/>
```

File: src/pages/embedded/containers/use-profiles.md (hunk 1 above)
Comment (nicolas-rabault) [reply src/pages/embedded/containers/use-profiles.md:44]: Yes *template container* is better.

**Review by Simonbdy — APPROVED** [review]
Looks good

**Pull request merged**